### ISSUE TITLE
is long filename omitted?

### ISSUE THREAD

**Issue by corselia** [lrsjng/h5ai#644]
h5ai is great product, thanks.

btw when some files with long filename are listed, each filename is shown as omitted. for example if original filename is `foobarfoobar`, shown filename is `foobarfoob...` .

I can't find this configure (about shown string count) in config file.

Can I change this config? if so, how to do?

**Comment by Armstrong1992** [comment]
Could you post relevant image? Or try changing: https://github.com/lrsjng/h5ai/blob/44e53ab213d37f45df2acac01e957c67ec06f8a7/src/_h5ai/public/css/lib/view/view.less#L56 to scroll or none.

**Comment by corselia** [comment]
@Armstrong1992 
oh! very^3 sorry for slow reply...

I try it as you, please wait a minutes

**Comment by corselia** [comment]
@Armstrong1992
I tried it. the result is below.

## prepare
I prepare two files.

- `ABCDEFGHIJKLMNOPQRSTUVWXYZ.jpg`
- `abcdefghijklmnopqrstuvwxyz_ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz_ABCDEFGHIJKLMNOPQRSTUVWXYZ.jpg`

## the image of three cases in `overflow` property

#### `hidden` (default)
<img width="698" alt="overflow_hidden" src="https://user-images.githubusercontent.com/14275619/34284025-63535b60-e712-11e7-8a35-dce04b83298c.png">

#### `scroll`
<img width="714" alt="overflow_scroll" src="https://user-images.githubusercontent.com/14275619/34284032-71e55f70-e712-11e7-8a95-769cced12d1c.png">

#### `none`
<img width="692" alt="overflow_none" src="https://user-images.githubusercontent.com/14275619/34284047-889dc842-e712-11e7-8294-552585e871df.png">

## wish
I wish the state of `overflow: scroll` and the filename isn't omitted.

thank you :smile:

**Comment by Armstrong1992** [comment]
Which version do you use? Dev or Release?
![1](https://user-images.githubusercontent.com/5490178/34287292-a0c632b4-e6e6-11e7-9e52-72ea2622d1c9.png)
![2](https://user-images.githubusercontent.com/5490178/34287294-a0e46f7c-e6e6-11e7-95de-1ab1b462911d.png)

**Comment by corselia** [comment]
I use version 0.29.0 release from [official site](https://larsjung.de/h5ai/).
direct link is https://release.larsjung.de/h5ai/h5ai-0.29.0.zip

**Comment by Armstrong1992** [comment]
Edit: public/js/lib/scripts.js and replace:
```js
.item .label {line-height: "+(e+14)+"px "+o+"
```
With:
```js
.item .label {line-height: "+(e+14)+"px "+o+"; overflow-x: auto

**Comment by corselia** [comment]
thanks. soon I tried.
so I noticed the behavior of `Chrome` and `Safari` are different from `Firefox`'s one.

in case both without and with modified `script.js`, the result were the same in each browser.
but Firefox showed the result I want.

#### the screenshot of Firefox
<img width="763" alt="the_screenshot_of_fx" src="https://user-images.githubusercontent.com/14275619/34288280-a1461c6a-e72f-11e7-9fcd-dc81428f27e5.png">

btw I tried not only `.item .label {line-height: "+(e+14)+"px "+o+"; overflow-x: auto` but also `.item .label {line-height: "+(e+14)+"px "+o+"; overflow-x: scroll`.

**Comment by Armstrong1992** [comment]
Replace also in public/css/styles.css:
```css
#view .label{display:block;overflow:hidden;text-align:left;text-overflow:ellipsis}
```
With:
```css
#view .label{display:block;overflow:hidden;text-align:left;}

And use ctrl+F5 to clear browser cache!

**Comment by corselia** [comment]
WOW! great and great!
showed correctly on `Chrome` and `Safari`!

summarize, I changed two descriptions in `styles.css` as below, both in `#view .label`
- from `overflow:hidden` to `overflow:scroll`
    - `overflow:auto` is also ok
- remove `text-overflow:ellipsis`

note:
- `scripts.js` keeps default.
- I checked with these web browsers
    - Safari (macOS)
    - Chrome (macOS and Android)
    - Firefox (macOS and Android)

very very thanks for your advice.
`h5ai` is really awesome product! 🎉
